hummingbot/client/config/config_var.py

```diff
@@ -6,14 +6,15 @@
 
 from typing import (
     Optional,
-    Callable
+    Callable,
+    Union,
 )
 import inspect
 
 # function types passed into ConfigVar
 RequiredIf = Callable[[str], Optional[bool]]
 Validator = Callable[[str], Optional[str]]
-Prompt = Callable[[str], Optional[str]]
+Prompt = Union[Callable[[str], Optional[str]], Optional[str]]
```

Comment:
The diff is weird, really all I did was add a `Union` of the existing type + `Optional[str]`